### PR TITLE
local.md updated

### LINKED ISSUE #453: typo in local installation guide
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

In the local installation guide
screenshot from 2018-01-21 14-15-05

**Expected behavior:**

It should be `sudo service mongodb start`

I would like to work on this issue.

### PR DESCRIPTION
**Pull request by nityanandagohain**
Fixes #453 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

if `sudo service mongod start` doesnot work then we should use `sudo service mongodb start`

### CODE REVIEW

**Comment by nityanandagohain** [comment]
@vaibhavsingh97 what should  I do now ? should I close this. Please help me

**Comment by cclauss** [comment]
You need to put more text around this change explaining the comments made above.  In general, we want to advise the user to try one command and if that does not work to try the other command.

File: docs/installation/local.md [hunk 1]
```diff
@@ -46,6 +46,12 @@ sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 7F0CEB10
 echo "deb http://repo.mongodb.org/apt/ubuntu "$(lsb_release -sc)"/mongodb-org/3.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-3.0.list
 sudo apt-get update
 sudo apt-get install -y mongodb
+sudo service mongod start
+```
+
+if `sudo service mongd start` throws "Failed to start mongod.service" error then execute
```
Comment (cclauss) [inline docs/installation/local.md]: mongod not mongd ;-)

Comment (nityanandagohain) [reply docs/installation/local.md]: sorry :smile:

**Comment by nityanandagohain** [comment]
@cclauss please check now :smiley:

File: docs/installation/local.md [hunk 2]
```diff
@@ -49,6 +49,12 @@ sudo apt-get install -y mongodb
 sudo service mongod start
 ```
 
+if `sudo service mongod start` throws "Failed to start mongod.service" error then execute
```
Comment (cclauss) [inline docs/installation/local.md]: Simplify to: if the final command raises a "__Failed to start mongod.service__" error then execute `sudo service mongodb start`

File: docs/installation/local.md [hunk 3]
```diff
@@ -49,6 +49,12 @@ sudo apt-get install -y mongodb
 sudo service mongod start
 ```
 
+If the final command raises a "Failed to start mongod.service" error then execute
```
[the remark this answers is not in the document]
Comment (vaibhavsingh97) [reply docs/installation/local.md]: This will give user a better idea. 
Cc @cclauss @realslimshanky

**Comment by nityanandagohain** [comment]
@vaibhavsingh97  Please check this ... I am using 2.6 but still I have  to use "mongodb" instead of "mongod"
screenshot from 2018-01-22 12-22-24

File: docs/installation/local.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (rupav) [reply docs/installation/local.md]: Yeah that is more informative. @nityanandagohain what do you Think?

[the remark this answers is not in the document]
Comment (bhaveshAn) [reply docs/installation/local.md]: I Agree. Please do the requested changes.

[the remark this answers is not in the document]
Comment (nityanandagohain) [reply docs/installation/local.md]: sure

**Comment by nityanandagohain** [comment]
I have made the changes please check @vaibhavsingh97 @cclauss @bhaveshAn

**Comment by vaibhavsingh97** [comment]
@nityanandagohain Strange, let me look into this 👍

**Comment by cclauss** [comment]
This PR and #459 seem to cover the same ground.  While neither PR has a great title ;-) , I prefer this one.

**Comment by realslimshanky** [comment]
@cclauss I agree with you, this PR explicitly tell about which version is supported with which command.

**Comment by mariobehling** [comment]
Closing this due to inactivity. Please reopen new PR if you want to resolve this.

**Pull request closed without merging**